docs/getting-started/configuring-your-repository.md

```diff
@@ -22,7 +22,7 @@ To configure your repository, follow these steps:
 
 ## 2. Configuring code patterns {: id="configuring-code-patterns"}
 
-[Configure the tools and code patterns](../repositories-configure/configuring-code-patterns.md) that Codacy uses to analyze your repository.
+[Configure the tools and code patterns](../repositories-configure/configuring-code-patterns.md) that Codacy uses to analyze your repository. If security is important for your team, review the [security monitor](../repositories/security-monitor.md) for a detailed overview of potential security issues on your repository.
```

Comment:
In this context, the idea is to leverage the security monitor to help the users turn on the security code patterns, and not so much review the security issues already found.

As such, an instruction like this would convey the meaning in a more straightforward way:

```suggestion
[Configure the tools and code patterns](../repositories-configure/configuring-code-patterns.md) that Codacy uses to analyze your repository. If security is important for your team, review the [security monitor](../repositories/security-monitor.md) to ensure that your configuration provides adequate detection of security issues.
```